ui/token/TokenPageTitle.tsx

```diff
@@ -12,13 +12,15 @@ import type { ResourceError } from 'lib/api/resources';
 import useApiQuery from 'lib/api/useApiQuery';
 import { useMultichainContext } from 'lib/contexts/multichain';
 import { getTokenTypeName } from 'lib/token/tokenTypes';
+import { Alert } from 'toolkit/chakra/alert';
 import { Tooltip } from 'toolkit/chakra/tooltip';
 import AddressMetadataAlert from 'ui/address/details/AddressMetadataAlert';
 import AddressQrCode from 'ui/address/details/AddressQrCode';
 import AccountActionsMenu from 'ui/shared/AccountActionsMenu/AccountActionsMenu';
 import AddressAddToWallet from 'ui/shared/address/AddressAddToWallet';
 import AddressEntity from 'ui/shared/entities/address/AddressEntity';
 import * as TokenEntity from 'ui/shared/entities/token/TokenEntity';
+import { Reputation } from 'ui/shared/entities/token/TokenEntity';
```

Comment:
```suggestion
import * as TokenEntity from 'ui/shared/entities/token/TokenEntity';
```

I would keep the component namespaced for the sake of better readability.